.travis.yml

```diff
@@ -14,9 +14,7 @@ script:
   - rustc --version
   - mkdir .cargo
   - echo 'paths = ["."]' > .cargo/config
-  - git clone -q --depth 50 -b pending https://github.com/rust-gnome/examples _examples
+  - git clone -q --depth 50 -b pending https://github.com/gtk-rs/examples _examples
   - cd _examples
   - ./build_travis.sh
-  - if [ "$GTK" = "3.18" ]; then FEAT="gtk_3_18"; fi
-  - echo $FEAT
-  - cargo build --features "$FEAT gtk/embed-lgpl-docs" --jobs 1
+  - cargo build --features "$FEATURES gtk/embed-lgpl-docs" --jobs 1
```

Comment:
Why? `build_travis.sh` already does a build?

Reply:
Indeed! I removed it.